src/Catalyst.Core.Lib/DAO/Converters/DfsHashByteStringConverters.cs

```diff
@@ -30,8 +30,10 @@ namespace Catalyst.Core.Lib.DAO.Converters
 {
     public class ByteStringToDfsHashConverter : IValueConverter<ByteString, string>
     {
+        // @TODO we should return a valid CID object rather than string.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)